src/main/java/edu/ucsb/cs156/gauchoride/services/SystemInfoServiceImpl.java

```diff
@@ -22,13 +27,39 @@ public class SystemInfoServiceImpl extends SystemInfoService {
   @Value("${app.showSwaggerUILink:false}")
   private boolean showSwaggerUILink;
 
+  @Value("${app.startQtrYYYYQ:20242}")
+  private String startQtrYYYYQ;
+
+  @Value("${app.endQtrYYYYQ:20243}")
+  private String endQtrYYYYQ;
+
+  @Value("${app.sourceRepo:https://github.com/ucsb-cs156/proj-gauchoride-s24-5pm-6}")
```

Comment:
The default should be this, without the team number.
```suggestion
  @Value("${app.sourceRepo:https://github.com/ucsb-cs156/proj-gauchoride}")
```

The reason is that this code needs to be mergeable back in to the main repo that holds the production code.

Reply:
Resovled in https://github.com/ucsb-cs156-s24/proj-gauchoride-s24-5pm-6/commit/f54b7a3ee56433c9b984e89e73de111f210ab586, thanks!